cmd/gcrane/cmd/list.go

```diff
@@ -28,28 +29,30 @@ func init() { Root.AddCommand(NewCmdList()) }
 // NewCmdList creates a new cobra.Command for the ls subcommand.
 func NewCmdList() *cobra.Command {
 	recursive := false
+	json := false
```

Comment:
wdyt about `--format=json` to sort-of match with #656 ?

Reply:
What other formats might there be?